### ISSUE TITLE
Member spotlight

### ISSUE THREAD

**Issue by usama9500** [Trusted-Seed/coordination#88]
From iteration0 created by [kristoferlund](https://github.com/kristoferlund): commons-stack/iteration0#1180  

Static content box that can be edited through CMS. Typical link will be to a Medium article or a YouTube vid.

## Details
- 

## Open questions
- Content and ideas for formatting needed @mafer-cordovas @rodricastillo9013 
  - Which members do we highlight
  - How do we present them?
    - Photo?
    - Quote??
    - ...

## 🔗 The links and docs related to this issue

**Comment by usama9500** [comment]
@usama9500 will help me with this task.

He created this document where we are going to have all the descriptions needed for this section: https://docs.google.com/document/d/1HKzSY_3iJzDPYRlYLuJebdFD7b-vVClupbsffmLFa9Y/edit?usp=sharing

**Comment by YineisyM** [comment]
Hi @kristoferlund I added this information directly in the Dashboard. I thought for a moment that clicking on the faces in the top 10 would redirect me to the link I decided to add, but then I realized that's not the case because that section is just: name, description and image. However, if we want to highlight a different member [here's the document](https://docs.google.com/document/d/1HKzSY_3iJzDPYRlYLuJebdFD7b-vVClupbsffmLFa9Y/edit?usp=sharing) with the details.

Should I close this issue?
